### DISCUSSION TITLE
Welcome to piler Discussions!

### DISCUSSION THREAD

**Discussion by jsuto** [jsuto/piler#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by Ramalama2** [comment]
Hey jsuto, amazing that you moved to github, makes everything more accessible for everyone :-)
Thank you!

**Reply by jsuto** [reply]
Indeed. It took me a while to accept that github offers more features than bitbucket.
